.gitignore

```diff
@@ -5,4 +5,5 @@ lerna-debug.log
 
 .eslintcache
 
-.idea
\ No newline at end of file
+.idea
+dummy
```

Comment:
todo: Remove
```suggestion
```

Reply:
I think it should be nice to be allowed to have a dummy directory without having to always being careful to not push it. This where for example, i put my tests launcher etc when debugging for the community and it allows me to not have to maintain another project wdyt? Maybe in another pr?

Reply:
Got it! Haven't had the need for such a directory myself, but I can see the use case. Though, It might require a couple of paragraphs in our contribution (or similar) with examples for it to make sense for others @adrien2p

Reply:
Maybe i could extract that to another pr where i would include the example cases in the contribution file the ? Wdyt?

Reply:
Like that idea! 👍

Reply:
Ahah true 🖖 where it is?🙈